# Dynmaic version detection does not work with pip install with --no-build-isolation
#
# @author stefan.wolfsheimer (SURF)


diff -Nru Cartopy-0.22.0.orig/pyproject.toml Cartopy-0.22.0/pyproject.toml
--- Cartopy-0.22.0.orig/pyproject.toml	2023-08-05 02:47:19.000000000 +0200
+++ Cartopy-0.22.0/pyproject.toml	2023-09-08 16:06:34.634999817 +0200
@@ -10,6 +10,7 @@
 
 [project]
 name = "Cartopy"
+version = "0.22.0"
 authors = [
     {name = "UK Met Office"},
 ]
@@ -46,7 +47,6 @@
     "pyshp>=2.1",
     "pyproj>=3.1.0",
 ]
-dynamic = ["version"]
 
 [project.optional-dependencies]
 doc = ["beautifulsoup4", "pydata-sphinx-theme", "sphinx", "sphinx-gallery"]
